tests/main/selinux-clean/task.yaml

```diff
@@ -0,0 +1,52 @@
+summary: Check that basic snap management does not raise any SELinux denials
+
+description: |
+    On systems where SELinux is supported, make sure that starting snapd and
+    performing basic install/remove tasks does not cause SELinux denials. Even
+    though we do not support SELinux for enforcing confinement of snaps, we do
+    not want to cause unnecessary warnings when users are performing basic
+    management tasks on snaps.
+
+systems: [fedora-*, centos-*]
+prepare: |
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+
+    # Install some fonts so that the fc-cache helpers have something to work with
+    distro_install_package fontconfig dejavu-sans-fonts
+
+    # Enable enforcing mode, our policy is already marked as permissive, so we
+    # will get audit entries but the program will not be stopped by SELinux
+    setenforce 1
+    ausearch --checkpoint stamp -m AVC || true
+
+restore: |
+    rm -f stamp
+    setenforce 0
+
+execute: |
+    snap install test-snapd-tools
+    test-snapd-tools.cmd echo 'hello world'
+    su -c 'test-snapd-tools.cmd echo hello world' test
+    snap remove test-snapd-tools
+    ausearch --checkpoint stamp --start checkpoint -m AVC 2>&1 | MATCH 'no matches'
+
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB/snaps.sh"
+
+    install_local test-snapd-service
```

Comment:
Can we also test `snap start ...` and `snap stop ...` here for this snap, or are these covered already elsewhere?

Reply:
Added

Reply:
Thanks!